e2e-tests/src/NPG/helper.js

```diff
@@ -0,0 +1,84 @@
+export const retrieveValidRedirectUrl = async (pmHost, paymentMethodId) => {
+  const urlGetUser = `https://portal.test.pagopa.gov.it/pmmockserviceapi/cd/user/get`;
+  const responseGetUser = await fetch(urlGetUser, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  if (responseGetUser.status === 200) {
+    const user = await responseGetUser.json();
+    const walletTokenCreditCard = user.sessionToken;
+    const urlPutUser = `https://portal.test.pagopa.gov.it/pmmockserviceapi/cd/user/save`;
+    const responsePutUser = await fetch(urlPutUser, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(user),
+    });
+    if (responsePutUser.status === 200) {
+      const urlStartSession = `${pmHost}/payment-wallet/v1/wallets`;
```

Comment:
pmHost this variable should be called walletHost

Reply:
I've added a different env varible called WALLET_HOST and renamed variables in the npg test and helper.